Source/Tx.Windows/PerfCounters/PerfCounterRealTimeProbe.cs

```diff
@@ -55,7 +55,10 @@ public void OnTimer(object state)
 
         public override void Dispose()
         {
-            _timer.Dispose();
+            if (_timer != null)
+            {
+                _timer.Dispose();
+            }
 
             base.Dispose();
```

Comment:
The proposed fix won't solve the issue since the base class [PerfCounterReader](https://github.com/Microsoft/Tx/blob/master/Source/Tx.Windows/PerfCounters/PerfCounteReader.cs) also has the similar issue, therefore it needs to be fixed as well ([here](https://github.com/Microsoft/Tx/blob/master/Source/Tx.Windows/PerfCounters/PerfCounteReader.cs#L88)) in order to solve the problem.